src/double_pendulum.sdf

```diff
@@ -0,0 +1,181 @@
+<?xml version='1.0' ?>
```

Comment:
why add the sdf, too?

Reply:
Original intent was to establish that I could load from SDFs too, but I agree that there's little point having both here, and the Val loading example looks similar enough to loading an SDF anyway in terms of setting up frames.